monkey/monkey_island/cc/services/agent_binary_service/build.py

```diff
@@ -73,11 +69,3 @@ def _log_agent_binary_hashes(agent_binary_repository: IAgentBinaryRepository):
 
     for os, binary_sha256_hash in agent_hashes.items():
         logger.info(f"{os} agent: SHA-256 hash: {binary_sha256_hash}")
-
-
-def _register_event_handlers(
-    island_event_queue: IIslandEventQueue, agent_binary_service: IAgentBinaryService
-):
-    island_event_queue.subscribe(
-        IslandEventTopic.SET_ISLAND_MODE, reset_masque_on_island_mode_change(agent_binary_service)
-    )
```

Comment:
This might cause an issue. If we hit the "clear all" button in the UI, does the masque get cleared? The UI should make an API call to clear it. Let's just make sure that's happening>

Reply:
Fixed in #3472